Added Tree_sort.py

hritik5102 opened the pull request:
Task complete @MAJOR-BEAST 

- [x] Added tree_Sort using python

if any changes required please let me know

Thank you :)

geekquad reviewed (changes requested):
Hey @hritik5102, I see that you have made 2 PRs (one here and the other one with the name of Kahns Algorithm). Added the same file in both of them. Please see to it and delete one accordingly. Changes needed:
* Adaboost needs to be in the path "ml/AdaBoost", not in "ml/Boosting/AdaBoost".
* I see that you have compared Ridge_Regression with Lasso Regression. Would you mind implementing Lasso Regression too?
* Please add the dataset used in the Dataset folder.
* Please make a requirements.txt file as well.
For any difficulty, feel free to contact me on Slack. Thank you.

hritik5102 commented:
> Hey @hritik5102, I see that you have made 2 PRs (one here and the other one with the name of Kahns Algorithm). Added the same file in both of them. Please see to it and delete one accordingly. Changes needed:
> 
> * Adaboost needs to be in the path "ml/AdaBoost", not in "ml/Boosting/AdaBoost".
> * I see that you have compared Ridge_Regression with Lasso Regression. Would you mind implementing Lasso Regression too?
> * Please add the dataset used in the Dataset folder.
> * Please make a requirements.txt file as well.
>   For any difficulty, feel free to contact me on Slack. Thank you.

@geekquad Check now, I have updated a requirement.txt for ridge regression and Adaboost, please check
